generic_config_updater/field_operation_validators.py

```diff
@@ -126,3 +126,71 @@ def _get_fields_in_patch():
                 return False
 
     return True
+
+
+def read_statedb_entry(table, field):
+    state_db = SonicV2Connector(host="127.0.0.1")
```

Comment:
>SonicV2Connector [](http://example.com/codeflow?start=15&length=16)

If you use Table class, you do not need to concat table name and key.

Reply:
I see.. is Table class recommended over SonicV2Connector use_unix_socket_path? In the future, this function may be extended to include tables other than PORT

Reply:
As I said, the benefit is no need to concat table name/separator/key. You can create multiple Table from one DBConnector, and they are not heavy.